pkg/network_test.go

```diff
@@ -675,3 +676,31 @@ func TestPortNumberForName(t *testing.T) {
 		})
 	}
 }
+
+func TestIsPotentialMeshErrorResponse(t *testing.T) {
+	for _, test := range []struct {
+		statusCode int
+		expect     bool
+	}{{
+		statusCode: 404,
+		expect:     false,
+	}, {
+		statusCode: 200,
+		expect:     false,
+	}, {
+		statusCode: 200,
```

Comment:
This is duplicated with above `statusCode: 200` test?

Reply:
Heh, thanks for the catch @nak3, fixed! 🤦‍♂️